src/SWP/Bundle/TemplateEngineBundle/Controller/ContainerController.php

```diff
@@ -221,7 +221,7 @@ public function linkUnlinkToContainerAction(Request $request, $id)
                     }
 
                     if ($position === false && $containerWidget) {
-                        throw new \Exception('Widget is already linked to container', 409);
+                        throw new \Exception('WidgetModel is already linked to container', 409);
```

Comment:
why `WidgetModel` when here we work with Container Widgets (it's relation between Widget and Container) no related to WidgetModel (storage for widget)